modules/client-configuration/pages/registration-overview.adoc

```diff
@@ -14,7 +14,13 @@ Before you begin, check that:
 
 [IMPORTANT]
 ====
-Do not register the {productname} Server to itself.
-The {productname} Server must be managed individually or by using another separate {productname} Server.
-For more information about using multiple servers, see xref:specialized-guides:large-deployments/hub-multi-server.adoc[].
+Do not register the {productname} Server base OS to SUSE Manager itself.
+The {productname} Server base OS must be managed individually or by using another separate {productname} Server.
+
+To manage {productname} Server Containers, use the [literal]``mgradm`` tool
+
+====
+[IMPORTANT]
+====
+After migrating from an older version of SUSE Manager to a newer version, we strongly recommend re-generating the existing bootstrap scripts before onboarding new systems to prevent any potential issues.
 ====
```

Comment:
```suggestion
====

ifeval::[{suma-content} == true]

[IMPORTANT]
====
After migrating from an older version of {productname} to a newer version, we strongly recommend re-generating the bootstrap scripts before onboarding new systems to prevent any potential issues.
====

endif::[]
```